service/controllers/messages.go

```diff
@@ -146,8 +176,8 @@ func routes(routeObject response, w http.ResponseWriter) {
 		w.Write(jData)
 		TextToSpeech(responseJSON.Message, 0)
 
-	} else if strings.ToLower(firstPars) == "weather" {
-
+	} else if strings.ToLower(firstPars) == "weather" || strings.ToLower(firstPars) == "see" || strings.ToLower(secondPars)=="at"||strings.ToLower(secondPars) == "weather"||strings.ToLower(thirdPars) == "at"{
```

Comment:
**see, at** ??

Reply:
see weather 
or 
see weather at 
shall i keep it or discard it?

Reply:
I am discarding it. It's of no use. And I am changing image to images